README.md

```diff
@@ -1,24 +1,83 @@
-# README
+# Database setup instructions
+Here we are going to follow the steps to configure our postgres database
+to work with our Rails application.
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+### 1. Make sure postgres is already installed on your machine.
+To install postgres on linux:
 
-Things you may want to cover:
+    $ sudo apt install postgresql postgresql-contrib
 
-* Ruby version
+To find the Postgres server version from the shell command line, simply issue a postgres command with the -V flag (for version):
 
-* System dependencies
+    $ postgres -V
+    postgres (PostgreSQL) 9.3.10
 
-* Configuration
+In the event that the postgres command is not found, you may need to locate the directory of the utility. This can be done by issuing the locate bin/postgres command:
 
-* Database creation
+    $ locate bin/postgres
+    /usr/lib/postgresql/9.3/bin/postgres
+Now with the direct path to the postgres utility, you can call it with the -V flag as illustrated above:
 
-* Database initialization
+    $ /usr/lib/postgresql/9.3/bin/postgres -V
+    postgres (PostgreSQL) 9.3.10
 
-* How to run the test suite
+### 2. Install the needed gem
+Located on our project folder we run the following command to install the needed gem to interact with postgres
 
-* Services (job queues, cache servers, search engines, etc.)
+    gem install pg
 
-* Deployment instructions
+If you get an error like this
 
-* ...
+    Building native extensions.  This could take a while...
+    ERROR:  Error installing pg:
+    ERROR: Failed to build gem native extension.
+    /home/User/.rvm/rubies/ruby-1.9.2-preview3/bin/ruby extconf.rb
+    checking for pg_config... yes
+    checking for libpq-fe.h... yes
+    checking for libpq/libpq-fs.h... yes
+    checking for PQconnectdb() in -lpq... no
+    checking for PQconnectdb() in -llibpq... no
+    checking for PQconnectdb() in -lms/libpq... no
+    Can't find the PostgreSQL client library (libpq)
+
+You need install the postgreSQL dev package with header of PostgreSQL. Run the following command
+
+    sudo apt-get install libpq-dev
+
+### 3. Add the postgres gem into Gemfile
+
+    gem 'pg', '~> 1.2.3'
+
+The 1.2.3 numbers are the current gem version, make sure you are adding the last version in your case.
+
+Run the bundle install to install the pg gem after adding that into Gemfile
+
+    bundle install
+
+### 4. Set up your database.yml with the code above
```

Comment:
You already added the changes in the database, meaning that other devs just need to set their en vars accordingly